server/src/test/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldTypeTests.java

```diff
@@ -40,4 +40,26 @@ public void testIsNotAggregatable() {
         MappedFieldType fieldType = new SparseVectorFieldMapper.SparseVectorFieldType(indexVersion, "field", false, Collections.emptyMap());
         assertFalse(fieldType.isAggregatable());
     }
+
+    public static SparseVectorFieldMapper.SparseVectorIndexOptions randomSparseVectorIndexOptions() {
+        if (randomBoolean() == false) {
+            return new SparseVectorFieldMapper.SparseVectorIndexOptions(false, null);
+        }
+
+        return new SparseVectorFieldMapper.SparseVectorIndexOptions(
```

Comment:
We could use a random boolean to determine whether token pruning config should be set vs null here too